src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -88,7 +88,7 @@ def get(self, request: Request, organization, tag_name) -> Response:
         projects = self.get_projects(request, organization)
         try:
             tag_values = get_tag_values(projects, tag_name, metric_names)
-        except InvalidParams as exc:
+        except (InvalidParams, DerivedMetricParseException) as exc:
```

Comment:
So you are suggesting not to actually catch the exception?
Hmm I feel like a ParseError is sensible here because theoretically the same misconfigured derived metric might not be misconfigured for another projects filter .. wdyt?

Reply:
OK that makes sense. I will think about this some more, but let's leave as-is for now 👍 .